src/styles/blocks/hamburger.scss

```diff
@@ -0,0 +1,13 @@
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  width: 18px;
+  height: 8px;
+  justify-content: space-between;
+
+  &__line {
```

Comment:
You can download an icon from figma so it's not really necessary to make this burger manually